Gems/Multiplayer/Code/CMakeLists.txt

```diff
@@ -197,6 +158,7 @@ if (PAL_TRAIT_BUILD_TESTS_SUPPORTED)
             NAMESPACE Gem
             FILES_CMAKE
                 multiplayer_tools_tests_files.cmake
+                multiplayer_tools_files.cmake
```

Comment:
these files are being compiled by Multiplayer.Editor and by this target, which means that they are being compiled twice. Instead, create a static library that builds multiplayer_tools_files and then link against it in the Editor module and this test

Reply:
took a second look and the number of files there is small so I will drop it, but this should still be done eventually to de-duplicate and reduce compilation time.

Reply:
Done.